client/control/components/RecipeList.js

```diff
@@ -169,6 +233,14 @@ class DisconnectedRecipeList extends React.Component {
   }
 }
 
+DisconnectedRecipeList.propTypes = {
```

Comment:
This was to satisfy a console error (something about propTypes being defined on an instance instead of on the class). This also conforms to the conventions used on other components.